scipy/optimize/optimize.py

```diff
@@ -2988,6 +2991,7 @@ class _Brute_Wrapper(object):
     """
     Object to wrap user cost function for optimize.brute, allowing picklability
     """
+
```

Comment:
It's not a good idea to make whitespace changes like this as it increases churn in a file, and makes it harder for other PR's because it sometimes creates merge conflicts.

Reply:
Sorry I did not see it, my editor automatically adds those white spaces... In the future I'll be more regarding.